system/jlib/jutil.cpp

```diff
@@ -1483,6 +1489,20 @@ unsigned usTick()
     gettimeofday(&tm,NULL);
     return tm.tv_sec*1000000+tm.tv_usec; 
 }
+#elif __APPLE__
+
+unsigned usTick()
+{
+    __uint64 nano = mach_absolute_time() * (uint64_t)timebase_info.numer / (uint64_t)timebase_info.denom;
```

Comment:
That would violate our previous assertion that get_cycles_now is only used for tracing (and thus it's ok for it to be unreliable on crappy hardware)